Sources/WebUI/WebView+Extension.swift

```diff
@@ -21,6 +21,8 @@ extension WebView: View {
         private func makeView() -> Remakeable<EnhancedWKWebView> {
             let webView = Remakeable {
                 EnhancedWKWebView(frame: .zero, configuration: parent.configuration)
+            } afterRemaking: {
+                parent.applyModifiers(to: $0)
             }
             setUpWebViewProxy(webView)
             parent.applyModifiers(to: webView.wrappedValue)
```

Comment:
I believe that the following code modification will also prevent the issue. This code modification has the advantage of not requiring any changes to the interface.

```suggestion
            let webView = Remakeable<EnhancedWKWebView> {
                let wrappedView = EnhancedWKWebView(frame: .zero, configuration: parent.configuration)
                parent.applyModifiers(to: wrappedView)
                return wrappedView
            }
            setUpWebViewProxy(webView)
```

Reply:
it's the best!

Reply:
@b1ackturtle 
I adopted your suggestion. [e240add](https://github.com/cybozu/WebUI/pull/20/commits/e240adde637d951072ad21158396aad38a5237b7)

Reply:
Cool.